src/ViewerComponents/Image/ImagesContainer.svelte

```diff
@@ -43,17 +44,42 @@
         // Checks if request succeeded
         if (!res.ok) {
             console.error(json);
-            return new Promise(() => {
-                throw new Error(`${res.status.toString()}: ${res.statusText}`);
-            })
+            return Promise.reject(
+                new Error(`${res.status.toString()}: ${res.statusText}`)
+            )
         }
 
         console.log(json);
         return json;
     }
 
     /**
-     * Checks if data is an empty object
+     * Requests the data count with the parameter given from parent component and the offset 0.
+     *
+     * @returns {Promise<unknown>}
+     */
+    async function requestDataCount() {
+        const res = await fetch(requestInfos['url'] + '/count', {
+            method: requestInfos['method'],
+            body: requestInfos['gravsearch'] + `\n OFFSET 0`
```

Comment:
OFFSET 0 doesn't change the return of a count request as far as i know. Since adding an offset other than 0 I would suggest to not add offsets when doing count requests

Reply:
I didn't know that